robocop/reports.py

```diff
@@ -9,13 +9,21 @@
 
     robocop --reports rules_by_id,some_other_report path/to/file.robot
 
-To enable all default reports use ``--reports all``.
+To enable all default reports use ``--reports all``.  Non-default reports can be only enabled using report name.
 
 The order of the reports is preserved. For example, if you want ``timestamp`` report to be printed before any
 other reports, you can use following configuration::
 
     robocop --reports timestamp,all src.robot
 
+Print list of all reports with their configured status by using ``--list-reports``::
+
+    robocop --reports all --list-reports
```

Comment:
Is it required to also call `--reports all` when the user just wants to list all the reports? Would be great to be able to just run `robocop --list-reports` and see all of them.

Reply:
It is not required. If you just run list reports it will run them all but they will have status (disabled). --reports all for only to show how configuration affects the output